syntax = "proto3";

package cakelab.arflow_grpc.v1;

import "cakelab/arflow_grpc/v1/session.proto";

option csharp_namespace = "CakeLab.ARFlow.Grpc.V1";

message ListSessionsResponse {
  repeated Session sessions = 1;
}
